src/cloudwatch/modules/dimensionplugins/generic_dimensions.py

```diff
@@ -0,0 +1,53 @@
+"""
+This is the file containing generic dimension plugin classes
+"""
+
+import os
+from . import DimensionPlugin
+
+"""
+This InstanceId Dimension coming from configured host instance
+"""
+
+def dimension_get_instance_id(dimension, args):
+    dimension[args['name']] = args['value']
+
+class Dimension_InstanceId(DimensionPlugin):
+    def register_plugin(self):
+        self.func = dimension_get_instance_id
+        self.args = {
+            'name': "InstanceId",
```

Comment:
looks like the name of this dimension was "Host", now it is changed to "InstanceId", do not know if it will bring confusion.

Reply:
This flexible dimension is configurable. We definitely can implement another Dimension_Host class with the same code. It is up to us and the user to expand the scenarios. Here, I am just giving two examples. I will write more document to explain how to use this feature.